Sources/ApiVideoPlayer/ApiVideoPlayerInformationNowPlaying.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import MediaPlayer
+
+protocol InformationNowPlaying {
+    var nowPlayingData: NowPlayingData? { get set }
+    func pause(currentTime: CMTime, currentRate: Float)
+    func play(currentTime: CMTime, currentRate: Float)
+    func updateCurrentTime(currentTime: CMTime)
+    func updatePlabackRate(rate: Float)
+}
+
+class ApiVideoPlayerInformationNowPlaying: InformationNowPlaying {
+    private var infos = [String: Any]()
+    private let taskExecutor: TasksExecutorProtocol.Type
+
+    init(taskExecutor: TasksExecutorProtocol.Type) {
+        self.taskExecutor = taskExecutor
+    }
+
+    var nowPlayingData: NowPlayingData? {
+        didSet {
+            guard let nowPlayingData = nowPlayingData else {
+                MPNowPlayingInfoCenter.default().nowPlayingInfo = [:]
+                return
+            }
+            if let title = nowPlayingData.title {
+                infos[MPMediaItemPropertyTitle] = title
+            }
+            infos[MPMediaItemPropertyPlaybackDuration] = nowPlayingData.duration.seconds
+            infos[MPNowPlayingInfoPropertyElapsedPlaybackTime] = nowPlayingData.currentTime.seconds
+            infos[MPNowPlayingInfoPropertyIsLiveStream] = nowPlayingData.isLive
+
+            infos[MPNowPlayingInfoPropertyPlaybackRate] = 1.0
+            #if !os(macOS)
+            if let thumb = nowPlayingData.thumbnailUrl {
+                if let url = URL(string: thumb) {
+                    updateRemoteArtwork(url: url)
+                }
+            }
+            #endif
+            MPNowPlayingInfoCenter.default().nowPlayingInfo = infos
+        }
+    }
+
+    func updateCurrentTime(currentTime: CMTime) {
+        self.overrideInformations(
+            for: MPNowPlayingInfoPropertyElapsedPlaybackTime,
+            value: currentTime.seconds
+        )
+    }
+
+    func updatePlabackRate(rate: Float) {
+        self.overrideInformations(
+            for: MPNowPlayingInfoPropertyPlaybackRate,
+            value: rate
+        )
+    }
+
+    func pause(currentTime: CMTime, currentRate: Float) {
```

Comment:
Do we have to pass `currentRate` and `currentTime` here?
The `currentRate` is already updated with `updatePlabackRate`.
I guess NowPlayingInfo infer the `currentTime` from the time the player is playing and the playback rate.

Only settings the `playbackState` should work. Right?
Same for play?